gobblin-modules/gobblin-http/src/main/java/gobblin/writer/AvroHttpWriterBuilder.java

```diff
@@ -36,4 +38,8 @@ public AvroHttpWriterBuilder fromConfig(Config config) {
     this.responseHandler = new HttpResponseHandler();
     return this;
   }
+
+  public String getLimiterKey (State state) throws NotConfiguredException {
+    return AvroHttpWriterBuilder.class.getName();
```

Comment:
The limiter key is actually the most critical part of the flow, as it defines how the policies will be set. The limiter key is the id of the resource for which we want to throttle access. In this case, we're not trying to throttle access to the writer, but to the http endpoint. The resource id should then be an identifier of the http endpoint, for example `httprequest/mydomain.com`, or `d2request/myServiceName`.